dgl_sparse/include/sparse/matrix_ops.h

```diff
@@ -26,6 +27,28 @@ namespace sparse {
 std::tuple<std::shared_ptr<COO>, torch::Tensor, torch::Tensor> COOIntersection(
     const std::shared_ptr<COO>& lhs, const std::shared_ptr<COO>& rhs);
 
+/**
+ * @brief Compact sparse matrix by removing rows or columns without non-zero
+ * elements in the sparse matrix and relabeling indices of the dimension.
+ *
+ * This function serves a dual purpose: it allows you to reorganize the
+ * indices within a specific dimension (rows or columns) of the sparse matrix
+ * and, if needed, place certain 'leading_indices' at the beginning of the
+ * relabeled dimension.
+ *
+ * @param mat The sparse matrix to be relabeled.
+ * @param dim The dimension to relabel. Should be 0 or 1. Use 0 for row-wise
+ *        relabeling and 1 for column-wise relabeling.
+ * @param leading_indices An optional tensor containing row or column ids that
+ *        should be placed at the beginning of the relabeled dimension.
+ *
+ * @return A tuple containing the relabeled sparse matrix and the index mapping
+ *         of the relabeled dimension from the new index to the original index.
+ */
+std::tuple<c10::intrusive_ptr<SparseMatrix>, torch::Tensor> Compact(
+    const c10::intrusive_ptr<SparseMatrix>& mat, uint64_t dim,
```

Comment:
```suggestion
    const c10::intrusive_ptr<SparseMatrix>& mat, int64_t dim,
```
Seems that Pytorch FFI doesn't have `uint64_t`.